saul-core/src/main/scala/edu/illinois/cs/cogcomp/saul/datamodel/DataModel.scala

```diff
@@ -245,7 +245,7 @@ trait DataModel extends Logging {
     def apply(f: T => List[String])(implicit tag: ClassTag[T], d1: DummyImplicit, d2: DummyImplicit, d3: DummyImplicit,
       d4: DummyImplicit, d5: DummyImplicit, d6: DummyImplicit): DiscreteCollectionProperty[T] = {
       def cachedF = if (cache) { x: T => getOrUpdate(x, f).asInstanceOf[List[String]] } else f
-      val a = new DiscreteCollectionProperty[T](name, cachedF, ordered) with NodeProperty[T] {
+      val a = new DiscreteCollectionProperty[T](name, cachedF, !ordered) with NodeProperty[T] {
```

Comment:
This is a bug that existed in the system from a long time ago and I noticed it when I was making these changes (also mention in #454). 
But fixing it required retraining systems, which at the time I didn't have time to do it. So I left it as an issue.